cleanlab/rank.py

```diff
@@ -556,52 +556,54 @@ def get_confidence_weighted_entropy_for_each_label(
     return label_quality_scores
 
 
-def get_knn_distance_ood_scores(
-    features: np.array, nbrs: NearestNeighbors, k: int = None
-) -> np.array:
-    """Returns the KNN distance out-of-distribution (OOD) score for each datapoint.
+def get_outlier_scores(features: np.array, knn: NearestNeighbors = None, k: int = 10) -> np.array:
+    """Returns the KNN distance outlier score for each datapoint.
 
-    This is a function to compute OOD scores where higher scores indicate the datapoint is more likely to be OOD.
+    This is a function to compute outlier scores where higher scores indicate the datapoint is more likely to be an outlier.
 
     Parameters
     ----------
     features : np.array
       Feature matrix of shape (N, M), where N is the number of datapoints and M is the number of features.
       This is the "query set" of features for each datapoint which are used for nearest neighbor search.
 
-    nbrs : sklearn.neighbors.NearestNeighbors
+    knn : sklearn.neighbors.NearestNeighbors
       Instantiated NearestNeighbors class object that's been fitted on a dataset in the same feature space.
       Note that the distance metric and n_neighbors is specified when instantiating this class.
```

Comment:
Add sentence:

You can also pass in a subclass of ` ``sklearn.neighbors.NearestNeighbors`` ` which allows you to use faster approximate neighbor libraries as long as you wrap them behind the same sklearn API.

Reply:
May also help to explicitly list exactly what methods of sklearn.neighbors.NearestNeighbors their custom class actually needs to support. I think it's only `fit()`, `kneighbors()`, and the`.n_neighbors` attribute, but double check this.